src/main/scala/fly/play/jiraExceptionProcessor/ReportingSupervisorStrategy.scala

```diff
@@ -42,15 +35,29 @@ class ReportingStrategyWrapper(wrapped: SupervisorStrategy, comment: String) ext
 
   private def reportException(context: ActorContext, exception: Throwable) = {
     val configuration = Configuration(context.system.settings.config)
-    val wsClient = {
-      val environment = Environment.simple(mode = Mode.Prod)
-      val wsConfig = new WSConfigParser(configuration, environment).parse
-      val ningWsConfig = new NingWSClientConfigParser(wsConfig, configuration, environment).parse
-      NingWSClient(ningWsConfig)
+
+    def withWsClient[T](code: WSClient => T): T = {
+      val wsClient = {
+        val environment = Environment.simple(mode = Mode.Prod)
+
+        val parser = new WSConfigParser(configuration, environment)
+        val config = AhcWSClientConfig(wsClientConfig = parser.parse())
+        val builder = new AhcConfigBuilder(config)
+        val ahcConfig = builder.configure().build()
+
+        implicit val system = ActorSystem("jiraExceptionProcessor")
```

Comment:
I was thinking, `context` (of type `ActorContext`) contains a `system` property which is an `implicit` `ActorSystem`, we could use that. What do you think is better? Use that or create new actor system as in the code above?